python/apps/taiga/src/taiga/stories/stories/services/__init__.py

```diff
@@ -235,7 +235,7 @@ async def _validate_and_process_values_to_update(
         status = await workflows_repositories.get_workflow_status(
             filters={"workflow_id": story.workflow_id, "id": status_id}
         )
-        if not status:
+        if not status or output.get("workflow", None):
```

Comment:
```suggestion
        if not status or output.get("workflow"):
```

Because
```python
>>> c = {"a": 1}
>>> c.get("a")
1
>>> c.get("b") == None
True
```